subt_ign/src/GameLogicPlugin.cc

```diff
@@ -1097,14 +1178,52 @@ void GameLogicPlugin::PostUpdate(
             // sim paused?
             if (dt <= 0)
               return true;
-
-            // Consider only velocity in the xy plane.
+            geometry_msgs::PoseStamped msg;
+            msg.header.stamp.sec = this->dataPtr->simTime.sec();
+            msg.header.stamp.nsec = this->dataPtr->simTime.nsec();
+	    msg.header.frame_id = "world"; // TODO: Convert to artifact_origin
```

Comment:
Convert robot poses and velocities so their frame is `artifact_origin` instead of the world origin

Reply:
Updated: e606acbe1c7532603535b556ac26ba069eaafc85